containers/keys/KeysTable.js

```diff
@@ -31,9 +32,12 @@ const KeysTable = ({ keys = [], onAction }) => {
                         <TableRow
                             key={ID}
                             cells={[
-                                <code key={1} className="mw100 inbl ellipsis">
-                                    {fingerprint}
-                                </code>,
+                                <div key={1} className="flex flex-row flex-nowrap flex-items-center">
+                                    <Copy value={fingerprint} />
```

Comment:
Make `<Copy>` smaller:
```suggestion
                                    <Copy value={fingerprint} className="pm-button--small" />
```

Reply:
Thought that was a bit big too, doesn't know about this class. Cool.